java/client/src/main/java/glide/api/models/commands/json/JsonArrIndexOptionsBinary.java

```diff
@@ -0,0 +1,46 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.json;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.commands.servermodules.Json;
+import glide.api.models.GlideString;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Additional parameters for {@link Json#arrindex} command. */
+public final class JsonArrIndexOptionsBinary {
```

Comment:
It is completely identical to `JsonArrIndexOptions`. It is safe to remove it - no strings used there.